src/OrleansCounterControl/OrleansCounterControl.csproj

```diff
@@ -45,8 +45,24 @@
     <WarningsAsErrors>4014</WarningsAsErrors>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.CodeAnalysis, Version=1.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Thanks for locating that comment link @sergeybykov

I agree that the project working practices should accurately reflect the real "rules" that are expected to be applied. I won't push back if you want to make those changes to the doc.
http://dotnet.github.io/orleans/Contributing